src/index.ts

```diff
@@ -53,6 +46,14 @@ const plugin: PluginImpl<Options> = (options = {}) => {
           ts.ScriptTarget.Latest,
           true, // setParentNodes
         );
+    } else {
+      // Rollup doesn't tell you the entry point of each module in the bundle,
+      // so we need to ask every TypeScript program for the given filename.
+      program = programs.find(p => (source = p.getSourceFile(fileName)));
+      if (!program && fs.existsSync(fileName)) {
```

Comment:
```suggestion
      if (!program && ts.sys.fileExists(fileName)) {
```

avoids the import, since we already use `ts.sys` in this file.